node/src/redis-modules/Json.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { RedisClient, RedisClusterClient } from "../../";
+import {
+    ConditionalChange,
+    JsonGetOptions,
+    createJsonGet,
+    createJsonSet,
+} from "../Commands";
+export type TRedisClient = RedisClient | RedisClusterClient;
+
+/**
+ * Sets the JSON value at the specified `path` stored at `key`.
+ * @param client - The Redis client to execute the command.
+ * @param key - The key of the JSON document.
+ * @param path - Represents the path within the JSON document where the value will be set.
+ * @param value - The value to set at the specific path, in JSON formatted string.
+ * @param setCondition - Set the value only if the given condition is met (within the key or path).
+ * Equivalent to ['XX' | 'NX'] in the Redis API.
```

Comment:
ConditionalChange already says it, lets remove this line so we won't need to maintain documentation here also (for example if another condition is added)